src/Voron/Impl/LowLevelTransaction.cs

```diff
@@ -87,6 +87,26 @@ public void Reset()
 
 
         public event Action<IPagerLevelTransactionState> BeforeCommitFinalization;
+
+        public long TotalEncryptionBufferSize
```

Comment:
Shouldn't `TotalEncryptionBufferSize` be part of `IPagerLevelTransactionState` (just next to `IPagerLevelTransactionState.CryptoPagerTransactionState`)?

Reply:
it is

Reply:
Right. I missed that.